src/Report_Upload.py

```diff
@@ -33,23 +45,98 @@ def browseFiles():
         
         # Change label contents
         label_file_explorer.configure(text="File Opened: "+filename)
-                                                                                            
-    # Create the root window
-    
+        button_submit = Button(frame,
+                        text = "Submit", command=sub)
+        button_submit.grid(column = 0,row = 5, columnspan=2)
+
     # Set window title
+    def sub():
+        if(repNameBox.get()!="" and patientIDBox.get()!=""):
+            report=repNameBox.get()
+            p_id=patientIDBox.get()
+            # p_id=str(p_id)
+            var1= "Patient ID: "+p_id +"\nName of the report: "+report
+            msg=tk.messagebox.askquestion("Are you sure?", var1 )
+            #if(msg=="yes"):
+            #db connectivity
+            copyReportFile()
+            addReports()
+        else:
+            var2="Please fill in the details"
+            msg1=tk.messagebox.showerror("ERROR", var2)
+
+    def copyReportFile ():
+        # source folder
+        global filename
+        # source filename
+        name = filename[filename.rfind('/')+1 : len(filename)]
+
+        # destination folder
+        newdir = pv.pdfPath
+        # destination filename
+        newfilename = repNameBox.get() + '.pdf'
+
+        newpath = newdir + '\\' + newfilename
+
+        shutil.copy2 ((filename), newdir)
+        os.rename ((newdir+ '\\' +name), (newdir+ '\\' +newfilename))
+
+    def addReports ():
```

Comment:
Just to be clear- in the db- you enter the name of the report right?
Last doubt, how do you get the file added to the assets directory?

Reply:
Yes, the report name gets added to the database, along with the report path (assets wala)

I use the shutil library for copying the file
The shutil.copy2() is used for that.